packages/workbox-routing/src/lib/router.js

```diff
@@ -19,14 +19,19 @@ import logHelper from '../../../../lib/log-helper.js';
 import normalizeHandler from './normalize-handler';
 
 /**
- * The Router takes one or more [Routes]{@link Route} and registers a [`fetch`
- * event listener](https://developer.mozilla.org/en-US/docs/Web/API/FetchEvent)
- * that will respond to network requests if there's a matching route.
+ * The Router takes one or more [Routes]{@link Route} and allows you to apply
+ * that routing logic to determine the appropriate way of handling requests
+ * inside of a service worker.
  *
  * It also allows you to define a "default" handler that applies to any requests
  * that don't explicitly match a `Route`, and a "catch" handler that responds
  * to any requests that throw an exception while being routed.
  *
+ * By default, the `Router` class won't register a `fetch` event handler,
```

Comment:
Nit: Could we rephrase to:

You can use the handleRequest() method to pass a fetch event through the router and ultimately get a "Routed response" back. If you'd like this to be handled automatically for you, calling `addFetchListener()` will cause the Router to respond to requests